pkgs/applications/editors/emacs/elisp-packages/manual-packages/idris2-mode/default.nix

```diff
@@ -20,10 +21,12 @@ melpaBuild rec {
     prop-menu
   ];
 
+  passthru.updateScript = gitUpdater { };
+
   meta = {
     homepage = "https://github.com/idris-community/idris2-mode";
     description = "Emacs mode for editing Idris 2 code";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ wuyoli ];
+    maintainers = with lib.maintainers; [ AndersonTorres ];
```

Comment:
What about co-maintaining?

From your commit message:
> wuyoli is inactive from at least one year

They created a [PR] last month.  I am a bit confused about "at least one year".



[PR]: https://github.com/NixOS/nixpkgs/pull/323443

Reply:
I was looking at the Git commit logs. Theey has only four commits: 

- one adding theemself as maintainer
- this package
- a fixup on river
- a fixup on avizo: 
```
  cce4df2373afdff8f9c53bd78841bc01cea474c3
  CommitDate: Sat Mar 18 00:56:49 2023 +0100
```

Reply:
Co-maintainership is OK, tho.